src/components/AddRecipe.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import FormLabel from '@mui/material/FormLabel';
+import TextField from '@mui/material/TextField';
+import useInputState from '../hooks/useInputState';
+
+function AddRecipe(props) {
```

Comment:
Not critical for this exercise, but in general we use `.js`/`.ts` for pure JavaScript/TypeScript files and `.jsx`/`.tsx` for JavaScript XML/TypeScript XML files like this one.

Reply:
Gotcha, fixing!

Reply:
Leave this as is if you want, it was just a comment so you know what's in our codebase 👍🏼